src/components/widgets/playlists/CreaterPlaylistheader.jsx

```diff
@@ -0,0 +1,118 @@
+import { Box, styled, Typography } from "@mui/material";
+import { Button } from "src/components/Button";
+import MusicIcon from "src/assets/svgs/music-icon.svg";
+import EditIcon from "src/assets/svgs/edit-icon.svg";
+import BinanceIcon from "src/assets/svgs/binance-icon.svg";
+import Dots from "src/assets/svgs/dots.svg";
+import { useState } from "react";
+import { BasicModal } from "src/layouts/app/Modal";
+
+const GridContainer = styled("div")({
+    display: "grid",
+    gridTemplateColumns: "repeat(12, 1fr)",
+    gridTemplateRows: "repeat(5, 1fr)",
+
+    position: "relative",
+    height: 360,
+    background: "#010C07",
+    cursor: "pointer",
+});
+
+const GridItemOne = styled("div")({
+    position: "absolute",
+    left: "36px",
+    top: "72px",
+});
+
+const GridItemTwo = styled("div")({
+    gridColumn: "5 / span 7",
+    gridRow: "3 / span 3",
+    paddingLeft: 72,
+});
+
+const GridItemThree = styled("div")({
+    gridColumn: "11 / span 2",
+    gridRow: "2 / span 1",
+});
+
+const Text = styled("p")(({ theme }) => ({
+    fontSize: 16.7,
+    color: theme.palette.tertiary.light,
+    marginRight: 24,
+}));
+
+export const CreaterPlaylistheader = () => {
+    const [isHover, setIsHover] = useState(false);
+    const onMouseEnter = () => setIsHover(true);
+    const onMouseLeave = () => setIsHover(false);
+
+    const [open, setOpen] = useState(false);
+    const handleOpen = () => setOpen(true);
+    const handleClose = () => setOpen(false);
+
+    return (
+        <Box>
+            <GridContainer onClick={handleOpen}>
+                <BasicModal onClose={handleClose} open={open} />
+                <GridItemOne onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
+                    {isHover ? <EditIcon /> : <MusicIcon />}
```

Comment:
Lemme tell you what I would have preferred here: The icons be used in a Paper/Box which styles them accordingly and adds the big circle background. But its fine all the same for now